Framework/test/produceNtuple_cfg.py

```diff
@@ -121,8 +121,8 @@
     branchName_jets_ak4 = cms.string('Jet'),
     branchName_jets_ak8_Hbb = cms.string('FatJet'),
     branchName_subjets_ak8_Hbb = cms.string('SubJet'),
-    branchName_jets_ak8_Wjj = cms.string('FatJetAK8LSLoose'),
-    branchName_subjets_ak8_Wjj = cms.string('SubJetAK8LSLoose'), 
+    branchName_jets_ak8_Wjj = cms.string('FatJet'),
```

Comment:
This change reminds me that at some point we should try lepton subject fractions (read from `FatJet_lsf3` branch) to figure out if boosted AK8 jets contain the selected lepton.